docs/greenboot.md

```diff
@@ -30,14 +30,26 @@ The health check script is installed into the `/etc/greenboot/check/required.d`
 directory and it is not executed during the system boot in case the `greenboot`
 package is not present.
 
+The `40_microshift_pre_rollback.sh` pre-rollback script is installed into the
+`/etc/greenboot/red.d` directory, to be executed right before the system rollback
+takes place. The script performs MicroShift pod and OVN database cleanup to avoid
+potential conflicts with previous versions of the software.
```

Comment:
Previous version may sound confusing, what about `conflicts with the version to be upgraded to`?

Reply:
Rephrased to make it sound less confusing.